bedrock/products/templates/products/vpn/platforms/android.html

```diff
@@ -41,124 +41,121 @@
-            <img src="{{ static('img/products/vpn/platforms/vpn-hero-mobile.png') }}" class="n-1" alt="" width="480" height="480">
-          </picture>
-        </div>
-      </div>
-    </div>
-  </section>
+  {% block hero %}
+    {% call vpn_platform_hero(
+      heading=ftl('vpn-shared-product-name'),
+      subheading=ftl('vpn-android-hero-headline'),
+      desc=ftl('vpn-android-page-description'),
+      image_url='img/products/vpn/platforms/vpn-hero-mobile.png'
+    ) %}
+      <a class="mzp-c-button mzp-t-xl" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-shared-platform-cta-button') }}</a>
+      <p class="guarantee-copy">
+        {{ ftl('vpn-shared-money-back-guarantee') }}
+      </p>
+    {% endcall %}
+  {% endblock %}
 
   <div class="mzp-l-content mzp-t-content-xl">
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-platforms-shared-privacy-headline'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-shared-platform-privacy-promise'),
       image_url='img/products/vpn/platforms/vpn-cntwell-privacy-promise.png',
-      class='vpn-content-well-right-half vpn-content-well-alt-layout'
+      class_name='vpn-content-media-right-half',
+      has_tight_leading=True,
+      has_outline=True,
+      loading='eager'
     ) %}
       <p>{{ ftl('vpn-android-maker-of-firefox') }}</p>
       <p>{{ ftl('vpn-android-ability') }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-platforms-shared-what-youll-get'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-shared-platform-what-youll-get'),
       image_url='img/products/vpn/platforms/vpn-cntwell-check-mark.png',
-      class='vpn-content-well-left-half'
+      class_name='vpn-content-media-left-half',
+      has_tight_leading=True,
     ) %}
-      <ul class="mzp-u-list-styled">
-        <li>{{ ftl('vpn-platforms-shared-feature-item-protection', devices=vpn_devices) }}</li>
-        <li>{{ ftl('vpn-platforms-shared-feature-item-access', countries=vpn_countries) }}</li>
-        <li>{{ ftl('vpn-platforms-shared-feature-item-encryption') }}</li>
-        <li>{{ ftl('vpn-platforms-shared-feature-item-bandwidth') }}</li>
-        <li>{{ ftl('vpn-platforms-shared-feature-item-logs') }}</li>
+      <ul class="mzp-u-list-styled vpn-feature-list">
+        <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-protection', devices=vpn_devices) }}</li>
+        <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-access-countries', countries=vpn_countries) }}</li>
+        <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-encrypt') }}</li>
+        <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-bandwidth') }}</li>
+        <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-activity-logs') }}</li>
+        <li class="vpn-feature-list-item"><a href="{{ url('products.vpn.landing')}}#faq-refunds">{{ ftl('vpn-shared-refund-policy') }}</a></li>
```

Comment:
Ooh I forgot to call this out. We should remove "Refund Policy" here as it's not really a product feature, and should only be accompanied underneath a subscribe button. We're making the same change on the main landing page to reflect this.

Reply:
Good call out. Removed!